### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by Gauvino**
I did a little bit of refactoring on the action workflow for easier PR in the future and fix some limiting problem, change the message for the stale issue and added a merge conflict labeler for people with merge conflict on their PR.

Fix some typo in prompt.

Check the change commit by commit please !

And for the version of the action use https://github.com/apps/renovate

### CODE REVIEW

File: .github/workflows/issue_stale.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+jobs:
+  issues:
+    name: Check for stale issues
+    runs-on: ubuntu-latest
+    if: ${{ contains(github.repository, 'BassT23/Proxmox') }}
+    steps:
+      - uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e # v9.0.0
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (Gauvino) [inline .github/workflows/issue_stale.yml:21]: Don't forget to match that !

**Comment by Gauvino** [comment]
Ready to merge !

File: .github/workflows/issue_stale.yml (hunk 1 above)
Comment (BassT23) [reply .github/workflows/issue_stale.yml:21]: Nice to see :)

please pull it to develop, not the stable ;)

**Comment by BassT23** [comment]
U use the master version for changing, would be better, you use the beta branch :)
There are a lot of changes :)

**Comment by Gauvino** [comment]
Good ?

**Comment by BassT23** [comment]
If you look in the conflicted files, I see you use the master branch.
Best is, if you look in the config file. This is totally recreated in beta.
If I now push the pull, all changes I made, are gone (in latest develop) then

**Pull request closed without merging**